index.html

```diff
@@ -0,0 +1,83 @@
+                <a href="login.html" class="login-button blue-button" alt="로그인 버튼">로그인</a>
+            </nav>
+        </header>
+
+        <section class="hero">
+            <div class="hero-content">
+                <div class="hero-left">
+                    일상의 모든 물건을<br>거래해 보세요
+                    <a href="items.html" class="blue-button">구경하러 가기</a>
+                </div>
+                <img src="images/Img_home_top.png" alt="판다"/>
+            </div>
+        </section>
+
+        <section class="part">
+            <div class="container">
+                <img src="images/Img_home_01.png" alt="hot-item"/>
+                <div class="text-content">
+                    <h2 class="part-name">Hot item</h2>
+                    <strong class="part-title">인기 상품을<br>확인해 보세요</strong>
+                    <p class="part-message">가장 HOT한 중고거래 물품을<br>판다 마켓에서 확인해 보세요</p>
+                </div>
+            </div>
+        </section>
+
+        <section class="part">
+            <div class="container">
+                <div class="search-text-content align-end">
+                    <h2 class="part-name">search</h2>
+                    <strong class="part-title text-right">구매를 원하는<br>상품을 검색하세요</strong>
+                    <p class="part-message text-right">구매하고 싶은 물품은 검색해서<br>쉽게 찾아보세요</p>
+                </div>
+                <img src="images/Img_home_02.png" alt="search"/>
+            </div>
+        </section>
+
+        <section class="part">
+            <div class="container">
+                <img src="images/Img_home_03.png" alt="register"/>
+                <div class="register-text-content">
+                    <h2 class="part-name">Register</h2>
+                    <strong class="part-title">판매를 원하는<br>상품을 등록하세요</strong>
+                    <p class="part-message">어떤 물건이든 판매하고 싶은 상품을<br>쉽게 등록하세요</p>
+                </div>
+            </div>
+        </section>
+
+        <section class="lower-part">
+            <div class="lower-part-container">
+                <div class="lower-part-message">믿을 수 있는<br>판다마켓 중고 거래</div>
+                <img src="images/Img_home_bottom.png"/>
+            </div>
+        </section>
+
+        <footer class="footer">
+            <div class="footer-container">
+                <div class="codeit-link">@codit - 2024</div>
+                <div class="privacy-policy-link">
+                    <a href="privacy.html">Privacy Policy</a>
+                    <a href="faq.html">FAQ</a>
+                </div>
+                <div class="images">
+                    <a href="/facebook.html" target="_blank"><img src="images/ic_facebook.png"></a>
+                    <a href="/twitter.html" target="_blank"><img src="images/ic_twitter.png"></a>
+                    <a href="/youtube.html" target="_blank"><img src="images/ic_youtube.png"></a>
+                    <a href="/instagram.html" target="_blank"><img src="images/ic_instagram.png"></a>
+                </div>
```

Comment:
💊 제안  
이 영역은 로고들의 모음이라고 할 수 있습니다. 관련된 요소들끼리 묶인 목록이니 의미록적인 마크업을 작성한다면 list 관련 태그로 작성하시는 것을 추천드립니다.

```suggestion
                <ul class="images">
                    <li><a href="/facebook.html" target="_blank"><img src="images/ic_facebook.png"></a></li>
                    <li><a href="/twitter.html" target="_blank"><img src="images/ic_twitter.png"></a></li>
                    <li><a href="/youtube.html" target="_blank"><img src="images/ic_youtube.png"></a></li>
                    <li><a href="/instagram.html" target="_blank"><img src="images/ic_instagram.png"></a></li>
                </ul>
```